umm-spec-lib/src/cmr/umm_spec/umm_g/granule.clj

```diff
@@ -105,4 +106,5 @@
      :Platforms (platform/PlatformRefs->umm-g-platforms platform-refs)
      :CloudCover cloud-cover
      :Projects (project/ProjectRefs->umm-g-projects project-refs)
-     }))
+     :TilingIdentificationSystem (tiling-system/TwoDCoordinateSystem->umm-g-tiling-identification-system
+                                  two-d-coordinate-system)}))
```

Comment:
Please do not move around the lines or parentheses. Just uncomment out the field that you are changing to minimize the merge conflict with other people who are also making changes to UMM-G.

Reply:
I think we should maintain proper clojure coding style with respect to parenthesis and spacing.

Reply:
This is a temporary thing. Once we added all the fields, we can do the proper formatting. 

Or a better way is if you make a version that works with parinfer and with the commented lines in between fixed lines (to minimize merge conflict), that would be great. I will go with that.